dashboard/test/factories.rb

```diff
@@ -393,6 +393,7 @@
         evaluator.section.try(:user) ||
         build(:teacher)
       follower.section = evaluator.section || build(:section, user: follower.user)
+      follower.user_id = follower.section.user_id
```

Comment:
Because though `follower.user` is an alias for `follower.section.user` as a result of the `has_one :user, through: :section`, empirically rails does not make `follower.user_id` an alias for `follower.section.user_id`.